projects/VitalRecord/URLs.cs

```diff
@@ -112,9 +112,6 @@ private string Trim(string url)
         /// <summary>URL for RelatedpersonBirthplace</summary>
         public string RelatedpersonBirthplace => Trim($"{_prefix}/StructureDefinition/Extension-relatedperson-birthplace-vr");
 
-        /// <summary>URL for RelatedPersonDeceased</summary>
```

Comment:
did you rerun the url script and that's how this was deleted? or was it done manually

Reply:
Reran the URL script